script/BaseData.s.sol

```diff
@@ -0,0 +1,215 @@
+/* solhint-disable no-console, gas-custom-errors, var-name-mixedcase */
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {Script} from "forge-std/Script.sol";
+
+contract BaseData is Script {
+    struct Data {
+        address OFT_OWNER;
+        address TOKEN_ADMIN;
+        address PROXY_ADMIN;
+        address LZ_ENDPOINT;
+        uint32 LZ_EID;
+    }
+
+    struct ChainIds {
+        uint256 mainnet;
+        uint256 base;
+        uint256 optimism;
+        uint256 arbitrum;
+        uint256 fraxtal;
+        uint256 manta;
+        uint256 taiko;
+        uint256 scroll;
+        uint256 fantom;
+        uint256 mantle;
+        uint256 blast;
+        uint256 linea;
+        // testnets
+        uint256 holesky;
+        uint256 sepolia;
+        uint256 fraxtalTestnet;
+        uint256 morphTestnet;
+    }
+
+    mapping(uint256 => Data) private __chainIdToData;
+
+    ChainIds private __chainIds = ChainIds({
+        mainnet: 1,
+        base: 8453,
+        optimism: 10,
+        arbitrum: 42161,
+        fraxtal: 252,
+        manta: 169,
+        taiko: 167000,
+        scroll: 534352,
+        fantom: 250,
+        mantle: 5000,
+        blast: 81457,
+        linea: 59144,
+        // testnets
+        holesky: 17000,
+        sepolia: 11155111,
+        fraxtalTestnet: 2522,
+        morphTestnet: 2810
+    });
+
+    address private TEMP_YN_SECURITY_COUNCIL;
+
+    function setUp() public virtual {
+        TEMP_YN_SECURITY_COUNCIL = makeAddr("temp-yn-security-council");
+
+        // NOTE: All the LZ Endpoints and EIDs are picked up from their docs
+        // at https://docs.layerzero.network/v2/developers/evm/technical-reference/deployed-contracts
+
+        // mainnets
+        __chainIdToData[__chainIds.mainnet] = Data({
+            OFT_OWNER: TEMP_YN_SECURITY_COUNCIL,
```

Comment:
why is this one set to TEMP_YN_SECURITY_COUNCIL ?

Should be the same security council

`0xfcad670592a3b24869C0b51a6c6FDED4F95D6975`